### ISSUE TITLE
Prefs styling problem (box hanging off the edge)

### ISSUE THREAD

**Issue by ghost** [nathan-osman/auchatstatus#40]
When going to the "Prefs" section I find that the "Retry interval" box is slightly off the dialog:

![sechatscriptprefs](https://cloud.githubusercontent.com/assets/11575271/11929653/04d884e4-a7d5-11e5-9822-a3751cd8b300.png)

I am running the latest version of your script on Firefox 43.

**Comment by nathan-osman** [comment]
Hmm... I couldn't reproduce on Firefox, but I'm guessing you are using a different theme that is causing the widgets to be a larger size.

**Comment by ghost** [comment]
Nope, I haven't installed any themes in Firefox, unless you mean system ones?

**Comment by nathan-osman** [comment]
Yeah, I was thinking of system themes.

**Comment by ghost** [comment]
Right, I am using the Numix theme. Do you need for me to be more specific than that or will that do?

**Comment by nathan-osman** [comment]
I'll have to either increase the fixed size of the box or come up with an algorithm for resizing it.

**Comment by ghost** [comment]
The latter sounds like a more dynamic solution, and probably a better one too.
